src/Features/Core/Portable/FeaturesResources.Designer.cs

```diff
@@ -436,18 +436,16 @@ internal class FeaturesResources {
                 return ResourceManager.GetString("Adding_an_imported_method_will_prevent_the_debug_session_from_continuing", resourceCulture);
             }
         }
-
+        
         /// <summary>
         ///   Looks up a localized string similar to Alias ambiguous type &apos;{0}&apos;.
         /// </summary>
-        internal static string Alias_ambiguous_type_0
-        {
-            get
-            {
+        internal static string Alias_ambiguous_type_0 {
+            get {
                 return ResourceManager.GetString("Alias_ambiguous_type_0", resourceCulture);
             }
         }
```

Comment:
someone manually edited this file in teh past.  my VS keeps fixing it, so i'm just going to merge this in since it's a PITA to have to undo this file every time.